### ISSUE TITLE
rose edit: crash in STASH requests panel

### ISSUE THREAD

**Issue by dpmatthews** [metomi/rose#1719]
Loading a UM 10.2 app, going to the STASH Requests panel and selecting `Packages -> Disable all packages` results in a crash.

**Comment by DrRakhi** [comment]
The same issue happens for us too. However not in Packages. Going to STASH Requests panel itself, rose edit crashes: showing following error: 
RuntimeError: could not create new GType: gl_um_fcst/widget/stash+StashSummaryDataPanelv1 (subclass of rose+config_editor+plugin+um+widget+stash+BaseStashSummaryDataPanelv1)

**Comment by arjclark** [comment]
@dpmatthews - is your error the same as in #1722?

**Comment by dpmatthews** [comment]
> @dpmatthews - is your error the same as in #1722?

I don't think they are related. The error I'm reporting appears to be easily repeated at our site and on several VMs. I've not been able to repeat the other error.

**Comment by arjclark** [comment]
Traceback for @dpmatthews's problem:

```
Traceback (most recent call last):
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/plugin/um/widget/stash.py", line 821, in <lambda>
    lambda i: self._packages_enable(disable=True))
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/plugin/um/widget/stash.py", line 874, in _packages_enable
    self.sub_ops.ignore_sections(sections_for_changing, disable)
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/ops/group.py", line 435, in ignore_sections
    is_ignored)
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/ops/group.py", line 345, in ignore_sections
    self.sect_ops.trigger_update(ns, skip_sub_data_update=True)
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/main.py", line 1088, in update_namespace
    self.updater.update_namespace(*args, **kwargs)
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/updater.py", line 210, in update_namespace
    self.update_sections(namespace)
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/updater.py", line 290, in update_sections
    config_name = self.util.split_full_ns(self.data, namespace)[0]
  File "/net/home/h06/aclark/rose.git/lib/python/rose/config_editor/util.py", line 79, in split_full_ns
    if full_namespace not in self.full_ns_split_lookup:
TypeError: unhashable type: 'list'
```

**Comment by benfitzpatrick** [comment]
Closed by #1835.
